aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -195,9 +303,13 @@ Resources:
       AutoScalingGroupName: {Ref: WebServer}
       MinSize: 2
       Recurrence: "0 3 * * TUE-SAT"
+<%   end -%>
```

Comment:
We can make manual changes on running resources without worry- only changes to the stack template will trigger changes on the running resources. So we can update the scheduled actions on the fly, and unless/until we change them in the stack template and push that change to production, they won't be changed back.